non-oss/spec/pharos_pro/phases/validate_version_spec.rb

```diff
@@ -1,7 +1,8 @@
-require 'pharos/phases/validate_version'
-require 'pharos_pro/phases/validate_version'
+describe Pharos::Phases::ValidateVersion, unless: Pharos.oss? do
+  before :all do
+    require 'pharos_pro/phases/validate_version'
```

Comment:
Hmm actually this and the require for the oss-phase may need to be `load`, otherwise it may become flaky based on spec run order.